java/integTest/src/test/java/glide/modules/BasicTests.java

```diff
@@ -0,0 +1,22 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.modules;
+
+import static glide.TestUtilities.commonClusterClientConfig;
+import static glide.api.models.configuration.RequestRoutingConfiguration.SimpleSingleNodeRoute.RANDOM;
+
+import glide.api.GlideClusterClient;
+import glide.api.models.commands.InfoOptions.Section;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+
+// TODO: remove when real tests added
+public class BasicTests {
```

Comment:
What's a basic test? 
```suggestion
public class VectorSearchTests {
```